src/css/typography.css

```diff
@@ -0,0 +1,13 @@
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+h2 {
+    font-size: 1.5rem;
+    font-weight: bold;
+}
+
+p {
+    font-size: 1rem;
+    line-height: 1.5em;
```

Comment:
Should this be `rem` to match the other sizes?

Reply:
Carl wanted the line height relative to the font-size, so it's em. If we later change font-size line-height with auto adjust.